voice/bxml/verbs/tag.md

```diff
@@ -0,0 +1,72 @@
+{% method %}
+## XML: `<Tag>`
+The Tag verb is used to set a new tag value without executing a callback. This new tag will be sent with
+all future callbacks unless overwritten by a future `tag` attribute or `<Tag>` verb, or cleared.
+
+### Text Content
+| Name | Description |
+|:-----|:------------|
+| tag  | The new tag value. Leading and trailing whitespace is trimmed. Leave blank to clear the tag. |
+
+### Callbacks Received
+
+None
+
+
+{% common %}
+#### Example 1 of 1: Redirect Verb
+This shows how to use BXML to set the tag for a call to the value "abc123" before playing an audio clip.
+When the platform hangs up at the end of the audio clip, the tag reported in the [disconnect event](../callbacks/disconnect.md)
+will be "abc123".
+
+{% sample lang="http" %}
+
+
+```XML
+<?xml version="1.0" encoding="UTF-8"?>
+<Response>
+   <Tag>
+      abc123
+   </Tag>
+   <PlayAudio>/myMediaFile.mp3</PlayAudio>
+</Response>
```

Comment:
What about if we had a tag before play audio being "mp3 was not played entirely" and a tag after the play audio with "mp3 was played"?
So in the disconnect callback we would know if the call was hung up early. :)

Reply:
Yeah I like that, just pushed an update